core/settings.py

```diff
@@ -129,14 +132,11 @@
 CATALOGUE_TOKEN = os.environ.get("CATALOGUE_TOKEN")
 ENV = os.environ.get("ENV")
 
-# session
-SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
 
-# Not actually used - Just required for LiveServerTestCase
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
-        "NAME": ":memory:",
+        "NAME": "db.sqlite3",
```

Comment:
Seems like the reason we rely on a database here is because [the backend inherits from ModelBackend](https://github.com/Weird-Sheep-Labs/django-azure-auth/blob/4ca256d0a3487094fcee9315dfba0d73284a8fc7/azure_auth/backends.py#L6) rather than [RemoteUserBackend](https://docs.djangoproject.com/en/5.0/ref/contrib/auth/#django.contrib.auth.backends.RemoteUserBackend)

It's possible there is another library that doesn't implement it this way, but I think it's quite likely we will need a database anyway at some point, so I'm not overly concerned about adding in a database dependency.

If we go down this path, then I assume we will need a follow up PR/ticket to configure Postgres rather than SQLite, so that in cloud platform multiple instances can share the same session store.

Reply:
Agreed, if we have more than one FE instance then we will need a shared DB.